i18n/hr.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@ctlaltdieliet is this expected?

cc/ @m1lt0n

Reply:
In server it's not normal. @ctlaltdieliet this file was removed in the past right? the format for the server is that an array is expected not an object (even if there are no translations in it), so validation of translations fails on server.

Reply:
It was removed but Weblate or a user recreated it. I removed it once more.
Once we get the user permissions right in Weblate, user will no longer be able to create languages in shipped.
Only admins will be able to do that.

Reply:
I pulled the latest commits @ctlaltdieliet and hr.json made its way again into the repo :/